themes/10up-theme/tailwind.config.js

```diff
@@ -10,32 +34,60 @@ module.exports = {
 	 */
 	content: [
 		/* Ensure changes to all PHP, JS, and JSON files rebuild your CSS */
-		'./assets/js/**/*.js',
-		'./includes/**/*.{php,js,json}',
-		'./partials/**/*.php',
-		'./templates/**/*.php',
+		'404.php',
+		'archive.php',
+		'attachment.php',
+		'author.php',
+		'category.php',
+		'date.php',
+		'footer.php',
+		'front-page.php',
+		'functions.php',
+		'header.php',
+		'home.php',
+		'index.php',
+		'page.php',
+		'search.php',
+		'searchform.php',
+		'single.php',
+		'singular.php',
+		'tag.php',
+		'taxonomy.php',
+
+		// Directories
+		'assets/**/*.js',
+		'includes/**/*.{js, php}',
+		'partials/**/*.php',
+		'templates/**/*.php',
 	],
 	safelist: [
-		/* Prevent editor-specific styles from being purged */
-		'editor-post-title__block',
-		'editor-post-title__input',
-		'entry-content',
-		'entry-title',
-		'block-editor-block-list__layout',
+		{
+			pattern: /(bg|text)-(primary|secondary|tertiary)/,
+		},
 	],
 	theme: {
 		/* Override the default theme */
-
+		container: {
+			center: true,
+			padding: {
```

Comment:
Do we want to override these values by default? It is quite an opinionated approach

Reply:
I'm sure everyone may have an opinion on this, but here's my thinking:

**Better DX**
After building 2-3 bigger projects, I think including the padding/centering like this as a default is way friendlier than having to add the centering/padding styles yourself. A couple of reasons for this:

It's way easier to just override the centering or padding then it is to add the equivalent styles each time.

Here's what it looks like to add the equivalent styles without that configuration:
```
<div class="container mx-auto sm:px-4 md:px-8 lg:px-10"> 
  ... 
</div>
```

Here's what it looks like to just completely remove the margin and padding while using the config I wrote:

```
<div class="container mx-0 px-0"> 
  ... 
</div>
```

Container styles come at the component layer so the utilities can easily override without having to account for every breakpoint that has padding by default. 

Also, the developer doesn't have to remember what container padding classes are needed by default, which helps promote consistency and reduce mental overhead (which are 2 of our larger objectives with the whole setup anyway). 

**Intuitive**
Virtually every single library/framework that uses a `container` abstraction has centering and padding. There are existing max-width utilities as well if someone just wants to have a particular maximum width at the upper end. So having this setup makes the container inherently more useful.

Reply:
@iansvo @dainemawer yeah I don't think we should get that opinionated with this. Although this is a nice way to cut down on some extra classes, it may not be always the approach we take to containers using responsive fixed widths